source/javascripts/_BitriseWorkflowEditor.js.erb

```diff
@@ -15,8 +15,9 @@ window.mode = "<%= mode %>";
 		"ng-showdown"
 	]);
 
-	BitriseWorkflowEditor.config(function($compileProvider) {
+	BitriseWorkflowEditor.config(function($compileProvider, $qProvider) {
 		$compileProvider.debugInfoEnabled(false);
+		$qProvider.errorOnUnhandledRejections(false);
```

Comment:
popup implemented using deferred-s and "no" is a reject. from angular 1.7 it is treated as errors by default although these are not real errors, so avoided this. (not the nicest solution but we should not spend time refactoring the angularjs codepart)